src/main/java/com/example/java/maven/gameBlackJack/Deck.java

```diff
@@ -17,6 +17,10 @@ public Deck() {
         Collections.shuffle(cards);
     }
 
+    public Stack<Card> getCards() {
```

Comment:
is it required? if so - return copy of stack instead of the same object you are storing in `deck` object

Reply:
Done